### PR TITLE
kops 1.4.1

### PR DESCRIPTION
**Pull request by WillPlatnick**
This commit adds a formula that installs kops, a tool for Kubernetes orchestration.

### CODE REVIEW

**Review by chrislovecnm — CHANGES_REQUESTED** [review]
You are going to need to depend on kubectl as well. I believe there is already a brew for it.

File: Formula/kops.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  def install
+    ENV["GOPATH"] = buildpath
+    kopspath = buildpath/"src/k8s.io/kops"
+    kopspath.install Dir["*"]
+
+    cd kopspath do
+      system "make"
+    end
+
+    cd buildpath do
```
Comment (MikeMcQuaid) [inline Formula/kops.rb]: This is unnecessary; you'll be `cd`d back there after you leave the `cd kopspath do` block.

File: Formula/kops.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  depends_on "go" => :build
+  depends_on "kubectl"
+
+  def install
+    ENV["GOPATH"] = buildpath
+    kopspath = buildpath/"src/k8s.io/kops"
+    kopspath.install Dir["*"]
+
+    cd kopspath do
+      system "make"
```
Comment (MikeMcQuaid) [inline Formula/kops.rb]: Alternatively, `system "make", "-C", kopspath` and skip the `cd`

File: Formula/kops.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    end
+
+    cd buildpath do
+      bin.install("bin/kops")
```
Comment (MikeMcQuaid) [inline Formula/kops.rb]: Is there no `make install` we can use?

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

File: Formula/kops.rb (hunk 3 above)
Comment (WillPlatnick) [inline Formula/kops.rb]: There is not. make installs the binary into GOPATH and leaves them there.

File: Formula/kops.rb (hunk 2 above)
Comment (WillPlatnick) [inline Formula/kops.rb]: Will push fix

File: Formula/kops.rb (hunk 1 above)
Comment (WillPlatnick) [inline Formula/kops.rb]: Will push fix

**Comment by WillPlatnick** [comment]
@MikeMcQuaid Requested changes addressed and squashed

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format is `kops 1.4.1 (new formula)`. Please complete the issue template in future PRs where this format is detailed in the linked [Contributing guidelines](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md). Thanks!

**Pull request closed without merging**